velox/common/base/SpillStats.h

```diff
@@ -141,3 +141,13 @@ void updateGlobalMaxSpillLevelExceededCount(
 /// Gets the cumulative global spill stats.
 SpillStats globalSpillStats();
 } // namespace facebook::velox::common
+
+template <>
```

Comment:
As we have moved to fmt 10 anyway, why not just use format_as and do away with the template specialization ?

Reply:
good point!